src/client/identity/identity.proto

```diff
@@ -0,0 +1,123 @@
+syntax = "proto3";
+
+package identity;
+option go_package = "github.com/pachyderm/pachyderm/src/client/identity";
+
+message IdentityConfig {
+	string issuer = 1;
```

Comment:
Normally in a dex server the issuer is configured with a config file at startup, but when you're initially setting up it can be tricky to get the right value. I pulled out this config and made the web server hot reloadable so users don't need to redeploy to iterate on the correct value.

Reply:
I renamed this to `IdentityServerConfig` - this is configuration for the Dex web server which handles OIDC requests.